src/main/java/io/github/opencubicchunks/cubicchunks/mixin/core/common/MixinServerChunkProvider.java

```diff
@@ -272,6 +277,16 @@ private void tickSections(CallbackInfo ci) {
                 this.level.getProfiler().push("broadcast");
                 ((ICubeHolder) cubeHolder).broadcastChanges(section);
                 this.level.getProfiler().pop();
+
+                // TODO do spawn radius check - need a cubic equivalent to `!this.chunkMap.noPlayersCloseForSpawning(chunkPos)`
+//                if (!this.chunkMap.noPlayersCloseForSpawning(chunkPos)) {
+                // TODO probably want to make sure column-based inhabited time works too
+                section.setCubeInhabitedTime(section.getCubeInhabitedTime() + timePassed);
+
+                // TODO mob spawning - refer to tickChunks implementation
+
+                ((IServerWorld) this.level).tickCube(section, randomTicks);
+//                }
```

Comment:
With this implementation columns are also ticked.
For extra compatibility only updating the inhabited time is important